_layouts/index.html

```diff
@@ -18,12 +18,6 @@
 						alt="Skip To Logo"></a>
 				<h1 id="project_title" style="display:inline; margin-left:10px;position:relative;top:-100px;">SkipTo
 					plugin by PayPal Accessibility Team</h1>
-				<section id="downloads">
-					<a class="zip_download_link" href="https://github.com/paypal/skipto/zipball/master">Download this
-						project as a .zip file</a>
-					<a class="tar_download_link" href="https://github.com/paypal/skipto/tarball/master">Download this
-						project as a tar.gz file</a>
-				</section>
```

Comment:
your call but I would prefer to fix the links which means making a release here on github which I can do

Reply:
I am not sure how these files are generated, but if the export the most current version, we can leave them in.

Reply:
I _think_ those links are to releases on github. I guess we will see when I build it. If not then I will remove the links

Reply:
I changed the link and these work now